gppro-google-webfonts.php

```diff
@@ -964,6 +964,9 @@ public function google_stack_list( $stacks = array() ) {
 
 				// If we dont already have the font, add it.
 				if ( ! isset( $stacks[ $type ][ $family ] ) ) {
+					if ( empty( $stacklist[ $family ] ) ) {
```

Comment:
Is this an unrelated change?

Reply:
huh, I didn't change that. I thought I made sure to start this from master, I wonder if this is one of those weird issues with git showing something already in the destination.